compiler+runtime/test/jank/form/case/pass-big-decimal.jank

```diff
@@ -0,0 +1,9 @@
+(assert
+ (=
+  (case 99999.9M
+    :no-match                             :no-match
+    99999.9M                              :big-nines
+    :default)
+  :big-nines))
+
+:success
```

Comment:
This whole test can just be a single `case`, without the `assert` or `=` or trailing `:success`.

```clojure
(case 99999.9M
  :no-match :no-match
  99999.9M :success
  :default)
```